.github/workflows/ci.yml

```
@@ -49,10 +48,16 @@ jobs:
         run: |
           flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
           flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
-      - name: Run tests
-        run: PYTHONPATH=src pytest --cov=experiment_generator --cov-report=xml
-      - name: Upload coverage report
-        uses: actions/upload-artifact@v4
+      - name: Run tests with coverage
+        run: |
+          python3 -m pytest -s \
+            --cov=./src \
```

Thanks for flagging. Missed this, but I think it should be:

```suggestion
            --cov=experiment_generator \
```